examples/README.md

```diff
@@ -10,6 +10,11 @@ You can install it with the following command:
 cargo install trunk wasm-bindgen-cli
```

Comment:
```suggestion
cargo install trunk
```
Can you include this fix as well? We don't need to install wasm-bindgen-cli manually anymore

Reply:
Nevermind, I can commit it myself